redirects.json

```diff
@@ -1 +0,0 @@
-{ "/contributing/donating": "/docs/contributing/promoting/" }
```

Comment:
Can you leave this file here? That way future editors can know that this feature exists? Or at least just comment this line out so that they know the format.

Reply:
sure! @graywolf336 done